.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,50 @@
+---
+name: 🐞 Bug Report
+about: Create a report to help us improve
+title: "[Bug] "
```

Comment:
This is the prepopulated title of the issue, which will make it easier to scan the issue queue for bugs vs. features.